include/deal.II/grid/tria_accessor.h

```diff
@@ -2535,6 +2535,57 @@ class CellAccessor :  public TriaAccessor<dim,dim,spacedim>
   unsigned int neighbor_face_no (const unsigned int neighbor) const;
 
   /**
+   * If the cell has a periodic neighbor at the given face, this function
+   * returns true, otherwise, the returned value is false. This function
+   * is valid for locally owned cells and ghost cells. For artificial cells,
+   * we do not store the periodicity data.
```

Comment:
Yes, this is in a newer commit. It seems that this is necessary for setting up the multigrid stuff, Martin was mentioning. Probably pairs for which one of the cells is artificial can be removed afterwards if this is desired.